NEWS.md

```diff
@@ -26,6 +26,7 @@
 * Improved wording of objects in signals and reports tab
 * Made table appearance for interactive and static tables consistent using new dependency `flextable`
 * Recodes NAs in signal tables to 'unknown' for all stratifications but not for the unstratified level
+* Added notification about negative values in age.
```

Comment:
same here as for the other PR. Can you move the added change to the new section of NEWS?

Reply:
Of course, I will do this.